ci/build-windows.yml

```diff
@@ -40,9 +40,10 @@ steps:
       ARTIFACT=daml-sdk-$(release_tag)-windows.tar.gz
       cp bazel-genfiles/release/sdk-release-tarball.tar.gz '$(Build.StagingDirectory)'/$ARTIFACT
       echo "##vso[task.setvariable variable=artifact;isOutput=true]$ARTIFACT"
-      WINDOWS_INSTALLER=daml-sdk-$(release_tag)-windows.exe
-      cp bazel-genfiles/release/windows-installer/daml-sdk-installer.exe '$(Build.StagingDirectory)'/$WINDOWS_INSTALLER
-      echo "##vso[task.setvariable variable=artifact-windows-installer;isOutput=true]$WINDOWS_INSTALLER"
+      WINDOWS_INSTALLER=daml-sdk-$(release_tag)-windows-unsigned.exe
+      WINDOWS_INSTALLER_PATH="$(Build.StagingDirectory)/$WINDOWS_INSTALLER"
+      cp bazel-genfiles/release/windows-installer/daml-sdk-installer.exe "$WINDOWS_INSTALLER_PATH"
+      echo "##vso[task.setvariable variable=artifact-unsigned-windows-installer;isOutput=true]$WINDOWS_INSTALLER"
```

Comment:
Maybe it's just me, but I found this piece of code very confusing. I'd recommend either removing the `WINDOWS_INSTALLER_PATH` variable (as it's only used once) or exporting it instead of the `WINDOWS_INSTALLER` one, as that's what you want to publish, and the publish step currently rebuilds that path.

Reply:
Agreed, that was leftover from when I still had the signing step included in the regular Windows step for testing purposes. I’ve removed it.